src/models/differentiable/evotree_model.jl

```diff
@@ -0,0 +1,147 @@
+using DataFrames
+using SliceMap
+using EvoTrees
+using MLJBase
+
+"""
+This type provides a basic interface to gradient-boosted tree models from the MLJ library.
+However, this is not be the final version of the interface: full support for EvoTrees has not been implemented yet and the `logits` and `probs` methods will be changed in the process of doing that if needed.
+"""
+
+"""
+    EvoTreeModel <: AbstractMLJModel
+
+Constructor for gradient-boosted decision trees from the EvoTrees.jl library. 
+"""
```

Comment:
It would be useful to mention what arguments you need to pass to create this object

Reply:
Sure, will fix.